app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -57,7 +58,13 @@ class MS.Views.TimelineTableView extends Backbone.View
 
   editStep: (e) ->
     e.preventDefault()
-    @stepModal.edit(@$(e.currentTarget).closest('[data-id]').data('id'), @refresh.bind(@))
+    @stepModal.edit(@stepIdFromEvent(e), @refresh.bind(@))
+
+  addLog: (e) ->
+    e.preventDefault()
+    unless @logModalView
+      @logModalView = new MS.Views.LogModalView(el: $("<div>").insertAfter(@$el))
+    @logModalView.showNew(@stepIdFromEvent(e), @refresh.bind(@))
```

Comment:
Everything follows the new pattern except this chunk. Why is @logModalView okay here?